TransCore/RPGCode.xml

```diff
@@ -2086,15 +2086,15 @@
 								{
 									label: (or (@ pageDesc 'acceptLabel)
 										(msnTranslate (@ options 'missionObj) 'AcceptLabel)
-										(typTranslate &dsRPGMission; 'actionAccept)
+										(typTranslate &dsDockScreenBase; 'actionAccept)
 										)
 									code: (@ pageDesc 'acceptCode)
 									nextPage: 'missionAccept
 									}
 								{
 									label: (or (@ pageDesc 'declineLabel)
 										(msnTranslate (@ options 'missionObj) 'DeclineLabel)
-										(typTranslate &dsRPGMission; 'actionDecline)
+										(typTranslate &dsDockScreenBase; 'actionDecline)
 										)
```

Comment:
I think it would be clearer to leave both of these as `typTranslate &dsRPGMission;` - it is the text for a mission screen afterall.  The actualy Text tags may be defined in `dsDockScreenBase` now, but they should be available in `dsRPGMission` via inheritance.